src/cmd/vendor.rs

```diff
@@ -758,7 +758,7 @@ pub fn copy_recursively(
 
         let filetype = entry.file_type()?;
         let canonical_path_filetype =
-            std::fs::metadata(std::fs::canonicalize(entry.path()).unwrap())
+            std::fs::metadata(std::fs::canonicalize(entry.path()).expect(format!("Failed to canonicalize {}.", entry.path().to_str().unwrap()).as_str()))
                 .unwrap()
```

Comment:
Would the following also properly print the error?
```suggestion
            std::fs::metadata(std::fs::canonicalize(entry.path())?)?
```

Reply:
This way, we do not have a panic in the code, but proper error handling. If additional information is not present in the above error output, we can do the following:
```suggestion
            std::fs::metadata(std::fs::canonicalize(entry.path()).map_err(|cause| {
                Error::chain(
                    format!("Failed to canonicalize {:?}.", entry.path().to_str()),
                    cause,
                )
            })?)?
```

Reply:
With first suggestion the output is following:

```
error: Cannot startup runtime. No such file or directory (os error 2)
```

which also does not give explicit information which file is the problem.

The second suggestion gives following:

```
error: Failed to canonicalize Some("/tmp/.tmp9r0Yc9/rtl/tb/dpi_models"). No such file or directory (os error 2)
```

which is already good.

With added unwrap to the entry name resolution, the output becomes following:

```
error: Failed to canonicalize "/tmp/.tmpLrfLD7/rtl/tb/dpi_models". No such file or directory (os error 2)
```